data/tutorials/gs_00_up_and_running.md

```diff
@@ -102,25 +106,25 @@ The second command (`eval $(opam env)`) modifies a few environments variables to
 
 **Please note:** After the `opam init` command, you might get a result asking if you'd like to update your `zsh` configuration. If you get that message, type in `N`, the default, then type `y` to install the hook, which will run `eval $(opam env)`. As you get more well-versed in OCaml, you can change these settings by rerunning `opam init`.
 
-Now check the installation by running `opam --version`. The current version is 2.1.2. Please note merely using `opam init` might install a previous version of opam. If you have a previous version, please refer to [the opam Upgrade Guide](https://opam.ocaml.org/doc/Upgrade_guide.html).
+Now check the installation by running `opam --version`. You can check if it's the current version on [opam.ocaml.org](https://opam.ocaml.org/). Please note merely using `opam init` might install a previous version of opam. If you have a previous version, please refer to [the opam Upgrade Guide](https://opam.ocaml.org/doc/Upgrade_guide.html).
```

Comment:
In fact, this line should be moved at the end of the `Installing opam` section. Indeed, it is for checking if opam has been installed, not for checking if it has been _initialized_. Moreover, it is normal not to have the latest released version of opam if it has been installed through the package manager, and the opam Upgrade Guide would not really help here...

```suggestion
Now check the installation by running `opam --version`. You can compare it with the latest released version on [opam.ocaml.org](https://opam.ocaml.org/).
```

Reply:
Both these sentence should be moved to `Installing `opam`` or just the second one?

Reply:
Thanks for the clarifications in Slack. Done!